src/Command/InstallCommand.php

```diff
@@ -401,6 +402,13 @@ protected function getDatabaseInformation(InputInterface $input, OutputInterface
             $dbPort       = $helper->ask($input, $output, $questionPort);
         }
 
+        if (strtolower($dbType) === "pgsql"
+            && ! $dbSslmode = $input->getOption('db-sslmode')
+        ) {
+            $questionSslmode = new Question('Please enter the connection\'s SSL mode (default: prefer): ', 'prefer');
```

Comment:
I want to release your PR with version `1.1.0` and this question backwards incompatible with old installer. I think we should replace the code block with something without interactive question for this option like:
```php
if (strtolower($dbType) === "pgsql" && ! $dbSslmode = $input->getOption('db-sslmode') {
    $dbSslmode = 'prefer';
}
```
If you want to set this option you can do it with `--db-pgsql-sslmode`.